### ISSUE TITLE
Replace Ads with Google Publisher Tags

### ISSUE THREAD

**Issue by thomaswilburn** [seattletimes/newsapp-template#6]
After doing a couple of rewrites with it, I think I've got a pretty good grasp on how GPT/DFP works, so we need to switch over new projects to use it instead (there's also a plan in the works to shim the old ad system for legacy pages across the site). 

Technically, this new system is "responsive" in the sense that we can decide which ads to show at different breakpoints. In reality, it still serves the old sizes, with the single exception that we can (technically) request a specific height for that one right-column ad (no more 600vs250px height wonkiness). But realistically, all the responsiveness means is that the ads don't show below a certain size, because we don't sell ads in actual responsive units.

@kbarlow I'm going to get this set into the template, but at some point we should site down and talk about how it will work in practice. We won't want to just hide them via CSS, but will still need to use a little bit of JS. We just need to decide whether we want that code located in the partials or in a new ads.js module.

**Comment by thomaswilburn** [comment]
Talked with Audrey, decided on something like this:

```
<%= t.include("partials/_ad.html", { type: "banner", hide: 720 }) %>
```

**Comment by thomaswilburn** [comment]
Fixed in 99f42017f3fd8bc846096f32e36f38c9a9df2c1b (mostly)
